src/test/kotlin/racingcar/RacingGameTest.kt

```diff
@@ -0,0 +1,49 @@
+package racingcar
+
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.inspectors.forAll
+import io.kotest.matchers.shouldBe
+import racingcar.domain.Car
+import racingcar.domain.RacingGame
+
+class RacingGameTest : ShouldSpec({
+
+    val tryCount = 10
+    val cars: List<Car> = listOf(
+        Car(name = "pobi"),
+        Car(name = "crong"),
+        Car(name = "honux")
+    )
+
+    beforeTest {
+        cars.forAll { it.location = Car.CAR_DEFAULT_LOCATION }
+    }
```

Comment:
테스트 픽스처를 만들어두고 매 테스트 케이스마다 초기화하여 재사용하고 계시군요? 👍💯

하지만 프로퍼티만 초기화하는 것보다는 객체 자체를 초기화하는 것이 조금 더 깔끔할 것 같아요. 😉

```suggestion
    lateinit var cars: List<Car>

    beforeTest {
        cars = listOf(
            Car(name = "pobi"),
            Car(name = "crong"),
            Car(name = "honux")
        )
    }
```

Reply:
이렇게  하고싶었는데 `lateinit` 키워드를 몰라서 강제로 초기화를 했었네요. ㅠㅠ
덕분에 좋은 키워드 공부해갑니다!